examples/interaction/tools/position_toolbar.py

```diff
@@ -1,3 +1,14 @@
+''' A basic scatter plot with the toolbar positioned below the plot,
+outside of the axes, titles, etc.
+This example shows a way to set the toolbar's location relative to the plot,
+outside of the axes, titles, etc.
```

Comment:
This second sentence seems a duplicative of the first. I'd suggest combining them into one sentence, or just dropping the second entirely.